lib/AST/Type.cpp

```diff
@@ -4145,6 +4159,10 @@ bool TypeBase::usesNativeReferenceCounting(ResilienceExpansion resilience) {
   llvm_unreachable("Unhandled type kind!");
 }
 
+bool TypeBase::usesNativeReferenceCounting(ResilienceExpansion resilience) {
```

Comment:
Is this used in enough places to justify its existence?

Reply:
No, it isn't. I'll fold it into callers.